recipe/meta.yaml

```diff
@@ -16,22 +16,20 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8
     - pip
   run:
-    - python >=3.6
+    - python >=3.8
     - astroid >=2.7
     - jinja2
     - pyyaml
-    - sphinx >=3.0
-    - sphinxcontrib-golangdomain
-    - sphinxcontrib-dotnetdomain
-    - unidecode
+    - sphinx >=6.1.0
+    - anyascii
 
 test:
-  requires:
-    - pydocstyle
-    - salvus_mesher_lite  # [osx]
+  # requires:
+  #   - pydocstyle
+  #   - salvus_mesher_lite  # [osx]
```

Comment:
Are these still needed? If not let's remove them.

Reply:
nope -- commented out while testing, and forgot to clean it up.

done now, and added myself as a maintainer for the future.